stratum/hal/lib/common/admin_utils.cc

```diff
@@ -2,7 +2,9 @@
 // Copyright 2018-present Open Networking Foundation
 // SPDX-License-Identifier: Apache-2.0
 
-#include <linux/reboot.h>
+#ifdef __linux__
+  #include <linux/reboot.h>
+#endif
```

Comment:
We also need to make a change in `AdminServiceUtilsInterface::Reboot()` below.

This might be a good candidate for the `glue/platforms.h` file. Define Reboot there and then we can call it here.